js/ftx.js

```diff
@@ -2130,6 +2134,9 @@ module.exports = class ftx extends Exchange {
             headers[keyField] = this.apiKey;
             headers[tsField] = timestamp;
             headers[signField] = signature;
+            if (chosenSubaccount !== undefined) {
+                headers['FTX-SUBACCOUNT'] = chosenSubaccount;
```

Comment:
We have to use the headerPrefix here, cause this integration actually works for two exchanges: ftx.com and ftx.us, that can be switched with the exchange.hostname: 

- https://github.com/ccxt/ccxt/blob/master/js/ftx.js#L21 . 

So, this header is actually `FTX-SUBACCOUNT` with ftx.com and `FTXUS-SUBACCOUNT` with ftx.us:

- https://github.com/ccxt/ccxt/blob/master/js/ftx.js#L346